access-control-app/int_test/cmr/access_control/int_test/access_control_group_crud_test.clj

```diff
@@ -205,7 +205,7 @@
 
         ;; Verify that user not in managing group does not have permission to add member to the group.
         ;; Remove grant-all ECHO acl from fixture
-        (e/ungrant (u/conn-context) {:id "guid4"})
+        (e/ungrant (u/conn-context) "ACL1200000001-CMR")
```

Comment:
agreed, but I want to hold off on this.  I noticed that new tests were added in the last few weeks which also hard code concept IDs.  This needs to be a wider effort across the code base.  Ill write an issue for it.